### PR TITLE
[CSP] Store static Cloudbeat configuration

### PR DESCRIPTION
**Pull request by uri-weisman**
## What does this PR do?
Stores config file per supported benchmark to be propagated to Cloudbeat.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## How to test this PR locally
Should be tested with an agent image holding a cloudbeat binary from the following branch:
- https://github.com/elastic/cloudbeat/pull/378

## Related issues
- Relates https://github.com/elastic/security-team/issues/4445

### CODE REVIEW

**Comment by ofiriro3** [comment]
Didn't go into too much detail!

But you rock! it looks awesome!

**Review by ofiriro3 — APPROVED** [review]
Very well done my friend looks good!

@JordanSh @orouz @CohenIdo 

Please go over this PR, we are making changes to the integration to support the new Cloudbeat configuration format.
If you have any objections, please set this PR as Request changes

@uri-weisman Please don't merge this PR until one of them will approve

**Comment by orouz** [comment]
@uri-weisman 

can we verify that installing this new integration still includes all relevant data (currently rules but now also fetchers?) in the [agent policy](https://github.com/elastic/integrations/pull/4168#issuecomment-1240836985)?

File: packages/cloud_security_posture/data_stream/findings/agent/stream/eks.yml.hbs [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: Findings
```
Comment (orouz) [inline packages/cloud_security_posture/data_stream/findings/agent/stream/eks.yml.hbs:1]: we might want to add validation to these files (if there isn't any?)

my concern is that this `yaml` lives in an `hbs` file so linting is harder and formatting issues might  be accidentally introduced, and if so, i _think_ we'd know about it only when it reaches cloudbeat

**Comment by uri-weisman** [comment]
> can we verify that installing this new integration still includes all relevant data (currently rules but now also fetchers?) in the [agent policy](https://github.com/elastic/integrations/pull/4168#issuecomment-1240836985)?

@orouz  - For an EKS integration

Screen Shot 2022-09-11 at 16 24 22

Screen Shot 2022-09-11 at 16 24 32

**Review by orouz — COMMENTED** [review]
LGTM

**Review by orouz — APPROVED** [review]

**Pull request merged**